functional/test.yml

```diff
@@ -16,16 +16,18 @@
     - assert: { that: "{{vault_status.status.sealed}} == False" }
     - assert: { that: "{{vault_status.rc}} == 0" }
 
```

Comment:
I think it would be easier to have a test_check.yml file that just tests that the write doesn't actually write